policy/enforcer.go

```diff
@@ -0,0 +1,126 @@
+package policy
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"sort"
+
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+
+	"github.com/castai/kvisor/castai"
+	"github.com/castai/kvisor/linters/kubelinter"
+	"github.com/samber/lo"
+	"golang.stackrox.io/kube-linter/pkg/k8sutil"
+	"golang.stackrox.io/kube-linter/pkg/lintcontext"
+	v1 "k8s.io/apiserver/pkg/apis/example/v1"
+)
+
+type Enforcer interface {
+	admission.Handler
+}
+
+type enforcer struct {
+	objectFilters []objectFilter
+	linter        *kubelinter.Linter
+}
+
+func NewEnforcer(linter *kubelinter.Linter) Enforcer {
+	return &enforcer{
+		objectFilters: []objectFilter{
+			skipObjectsWithOwners,
+		},
+		linter: linter,
+	}
+}
+
+func (e *enforcer) Handle(ctx context.Context, request admission.Request) admission.Response {
+	// Unmarshal object.
+	var object k8sutil.Object
+	kind := request.Kind.Kind
+
+	switch kind {
+	case "Pod":
```

Comment:
Some objects are missing, we have full list of objects which we lint here https://github.com/castai/kvisor/blob/main/linters/kubelinter/subscriber.go#L49

Reply:
but we do not have the rest of them as resources in our backend, how we will toggle enforcement for them?

Reply:
hmm, this list is missing cronjob and job...